index.js

```diff
@@ -8,6 +8,10 @@ var colors = require('colors');
 var chalk = require('chalk');
 var path = require('path');
 
+function getThemeName(theme) {
+  return theme.match('jsonresume-theme-.*') ? theme : `jsonresume-theme-${theme}`;
+}
```

Comment:
If `getThemeName` could be factored into a [custom options processor](https://www.npmjs.com/package/commander#custom-option-processing), the value would be computed once and stored on `program`. This way, `getThemeName` would not need to be invoked on lines 48 and 57 below.

Reply:
Good idea, will give it a try 👍

Reply:
Done.